src/formats/PDB.cpp

```diff
@@ -181,6 +191,18 @@ void PDBFormat::read_ATOM(Frame& frame, const std::string& line,
         );
     }
 
+    if (atom_offsets_.size() == 0) {
+        try {
+            auto initial_offset = std::stoul(line.substr(6,5));
+            if (initial_offset == 0) {
+                throw format_error("0 is not a valid atom id.");
+            }
+            atom_offsets_.push_back(initial_offset - 1);
+        } catch(std::invalid_argument&) {
+            throw format_error("{} is not a valid atom id.", line.substr(6,5));
```

Comment:
I would rather send a warning here, and try to continue parsing assuming 1 as atom id.

Reply:
This would allow us to continue parsing files with more than 9999 atoms, where there is no clear way of defining atom ids.

Reply:
I think using an integer for the `initial_offset` would be best, that way if someone is crazy enough to have a negative index, nothing goes crazy.